app/models/miq_worker/runner.rb

```diff
@@ -266,7 +266,6 @@ def sync_config
     Vmdb::Settings.reload!
     @my_zone ||= MiqServer.my_zone
     sync_worker_settings
-    sync_blacklisted_events
```

Comment:
Wild that this was in the base `MiqWorker::Runner` class :shrug:

Reply:
wow, this must be old... I wonder what non-event workers would need to sync blacklisted events?

Reply:
I think it was just implemented globally, because reasons - I don't recall it being for anything but event catchers in practice.

Reply:
Yeah I don't see any rationale in the original PR for why it was in the base base worker.  The sync_blacklisted_events method was added to the `BaseManager::EventCatcher::Runner` in an earlier commit in the same PR then added to the base worker towards the end